metals/src/main/scala/scala/meta/internal/parsing/FoldingRangeExtractor.scala

```diff
@@ -31,10 +26,9 @@ final class FoldingRangeExtractor(
   }
 
   def extractFrom(tree: Tree, enclosing: Position): Unit = {
-    if (span(tree.pos) > spanThreshold) {
+    if (span(tree.pos) > 0) {
```

Comment:
Let's revert the change if possible.

Reply:
This filter is now applied in `Foldable::unapply` so it's still in place. It just now gets overriden by expression being a `Defn`.

Reply:
Do we need it here still then?

Reply:
True here it's not needed, but we'll still need something to stop oneline `Defn`s from producing folding regions.